apps/admin_audit/lib/Actions/Action.php

```diff
@@ -31,12 +31,8 @@
 use OCA\AdminAudit\IAuditLogger;
 
 class Action {
-	/** @var IAuditLogger */
-	private $logger;
 
-	public function __construct(IAuditLogger $logger) {
-		$this->logger = $logger;
-	}
+	public function __construct(private IAuditLogger $logger) {}
```

Comment:
```suggestion
	public function __construct(
		private IAuditLogger $logger,
	) {}
```
This code style makes diff prettier and properties more obvious.

Reply:
Thanks, in future Pull requests I'll use these code styles.